qt/app.py

```diff
@@ -880,13 +881,9 @@ def updateProfiles(self):
 
         self.comboProfiles.clear()
 
+        update_combo_profiles(self.config, self.comboProfiles, self.config.currentProfile())
```

Comment:
```suggestion
        utils.update_combo_profiles(self.config, self.comboProfiles, self.config.currentProfile())
```
Explicit is better than implicit; Python Zen rule number 2.

The code-base of BIT is complex and smelly enough. It is hard to understand and to dive into it.

Reply:
agree